BranchMonsterFactory/ExternalLibs/Branch-SDK/BNCConfig.h

```diff
@@ -1,34 +0,0 @@
-//
```

Comment:
I do not remember doing anything to remove the ExternalLibs stuff. I don't know what that is. It's hluge though.